src/exp_and_gram.jl

```diff
@@ -37,16 +37,40 @@ function exp_and_gram!(
     return Φ, G
 end
 
+alloc_mem(A, B, method::ExpAndGram{T,q}) where {T,q} = nothing
+function alloc_mem(A, B, method::ExpAndGram{T,13}) where {T}
```

Comment:
I am currently agnostic. Perhaps `Tuple/NamedTuple` is fine for now. If there are issues down the road I guess it can be moved into a struct without code breakage.

Reply:
Then let's stick to the NamedTuple for now, and change later if necessary